src/cswinrt/strings/additions/Windows.Storage.Streams/WellKnownStreamInterfaceIIDs.cs

```diff
@@ -0,0 +1,45 @@
+namespace ABI.Windows.Storage.Streams
+{
+    internal static class WellKnownStreamInterfaceIIDs
+    {
+        public static ref readonly Guid IID_IBufferByteAccess
```

Comment:
All these are already in `WellKnownWindowsInterfaceIIDs`. We should add the missing ones to `WellKnownInterfaceIIDs` (e.g. `IID_IBufferByteAccess`), and then we can delete these here and just use the ones from the runtime instead.